src/sympc/protocol/falcon/falcon.py

```diff
@@ -63,3 +64,100 @@ def __eq__(self, other: Any) -> bool:
             return False
 
         return True
+
+    @staticmethod
+    def mul_master(x, y, session) -> ReplicatedSharedTensor:
```

Comment:
Could you add the type annotations in function for `x`,`y`,`session` definition similar to the docstring.